### DISCUSSION TITLE
💡 | New Option: Limit devices connected to the network

### DISCUSSION THREAD

**Discussion by COD3BreakerIX** [Mygod/VPNHotspot#539] [Ideas]
Hello @Mygod. Nice to meet you.

I apologize for bothering you a bit, but I got a very big idea for your application.

I would like if you could implement the following idea I have:

When sharing VPN via hotspot, in the "Clients" section, when a device appears, there is an option to block its access, right?

My idea would be that, instead of just blocking and unblocking the connecting device, I would like to be able to "Limit" devices using an adjustable switch, from 0 to 100, which performs the following function:

With the switch of the device at 0, it should block access as it normally does. However, if set to, for example, 25%, then the hotspot only shares 25% of the provided internet.
At 50%, it shares 50% of the internet.
At 100%, it shares 100% (meaning the receiving device is not blocked).

This is the idea:
![photo_2024-02-21_03-15-28](https://github.com/Mygod/VPNHotspot/assets/127803467/c8dbf821-c8b7-4002-aadc-ea899e2109c1),

![photo_2024-02-21-04-09](https://github.com/Mygod/VPNHotspot/assets/127803467/b15d9570-48ff-4edf-bf5b-e01a0a2c701a)

It would be a very good option, which could add a significant advantage. If you could implement it in your application, it's the only thing I would really like, and it would completely change the way of controlling connected devices, even though your application is already incredible with a lot of options that help me share the VPN signal 🙏

I apologize for the inconvenience, and I hope you like my idea, and perhaps you can implement it.

Thank you very much.

**Comment by Mygod** [comment]
#93
